### DISCUSSION TITLE
Ghritak's Introduction

### DISCUSSION THREAD

**Discussion by ghritak** [Open-Sustainability-Index/open-sustainability-index-backend#15] [General]
**Hi everyone!**

I’m Ghritak, and I’m excited to be joining this project. I’m a  software developer with experience in  JavaScript, React, React Native, Next.js, Node.js etc. I’m particularly interested in fixing bugs, developing new features.

I’ve been following this project for a while I’m looking forward to contributing and learning from this great community.

Thanks!

**Comment by achaido** [comment]
Hei there Ghritak! You're more than welcome. I think your best, most valuable source of help initially is @tomsoderlund and @hankmander who's main contributors. Please reach out to any of them with question, suggestions or help setting up the environment. We're pleased to have you onboard!  🙏 Really looking forward to seeing our contribution!

**Reply by ghritak** [reply]
Thank you so much @achaido

**Comment by tomsoderlund** [comment]
Awesome @ghritak! You are so welcome! And thank you for your first pull request.

How did you find out about OSI?

**Reply by ghritak** [reply]
Thanks @tomsoderlund , I discovered this project through GitHub’s Explore feature.
